pkg/rancher-components/src/components/Form/LabeledInput/LabeledInput.vue

```diff
@@ -105,6 +105,15 @@ export default defineComponent({
     class: {
       type:    String,
       default: ''
+    },
+
+    /**
+     * Optionally use this to comply with a11y IF there's no label
+     * associated with the input
+     */
+    accessibilityLabel: {
+      type:    String,
+      default: ''
```

Comment:
nitpick: I'm not much of a fan of `accessibilityLabel` as a prop. I think that we can just name this `aria-label` because I believe that we would want to control where one is applied if a developer tries to supply one to the component.